worldedit-core/src/main/java/com/sk89q/worldedit/command/UtilityCommands.java

```diff
@@ -515,17 +514,22 @@ public void remove(Actor actor, CommandContext args) throws WorldEditException,
         desc = "Evaluate a mathematical expression"
     )
     @CommandPermissions("worldedit.calc")
-    public void calc(Actor actor, @Text String input) throws CommandException {
-        try {
-            Expression expression = Expression.compile(input);
-            actor.print("= " + expression.evaluate());
-        } catch (EvaluationException e) {
-            actor.printError(String.format(
-                    "'%s' could not be parsed as a valid expression", input));
-        } catch (ExpressionException e) {
-            actor.printError(String.format(
-                    "'%s' could not be evaluated (error: %s)", input, e.getMessage()));
-        }
+    public void calc(final Actor actor, @Text final String input) throws CommandException {
+        // Dev should have threaded it years ago to prevent the exploit from happening.
```

Comment:
Cut the snark, comments like this won't make it into the final and makes it less likely they will pull your PR.